coralproject/talk
# [next] Broadcast mutation events

## What does this PR do?
- Broadcast mutation events to EventEmitter
- Relay to TalkEmbedStream
- https://www.pivotaltracker.com/story/show/162022986

## How do I test this PR?
- Run `TalkStreamEmbed.on("mutation.*", function (...args) {console.log(this.event, args)})` in console and see events logging to console when performing mutations
